src/lib/components/billing/selectPaymentMethod.svelte

```diff
@@ -85,22 +97,33 @@
         </svelte:fragment>
     </InputSelectSearch>
 {:else}
-    <Card
-        isDashed
-        style="--p-card-padding:0.75rem; --p-card-bg-color: transparent; --p-card-border-radius: 0.5rem"
-        isTile>
-        <div class="u-flex u-main-space-between u-cross-center">
-            <p class="u-flex u-gap-8 u-cross-center">
-                <span
-                    class="icon-exclamation u-color-text-warning"
-                    style:font-size="var(--icon-size-medium)"></span>
-                <span class="text">A payment method is required</span>
-            </p>
-            <Button secondary on:click={() => (showPaymentModal = true)}>
-                <span class="icon-plus"></span> <span class="text">Add</span>
-            </Button>
-        </div>
-    </Card>
+    <div>
+        <input
+            bind:this={input}
+            on:invalid={handleInvalid}
+            required
+            class="u-hide"
+            type="text"
```

Comment:
Maybe here we can use type=hidden

Reply:
![image](https://github.com/appwrite/console/assets/45005369/20177649-e892-453e-afdd-3b62d0356027)
We can't... I tested it and the hidden input doesn't have validity, which means we can't show the message

I think it's due to this:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/hidden#validation